WordPress/Docs/WP/DiscouragedConstantsStandard.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="Discouraged Constants"
+    >
+    <standard>
+    <![CDATA[
+    The usage of WordPress Constants is discouraged.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: A function is used in the code to retrieve the theme directory.">
+        <![CDATA[
+<img src="
```

Comment:
AFAIK, it is not common to indent an HTML tag like it is done here. Maybe instead of fixing the indentation, we can simplify the example and make it easier to understand what this sniff is about? I believe the code sample could be trimmed down to `echo <em>TEMPLATEPATH</em>;`.

Reply:
Actually just `echo <em>TEMPLATEPATH</em>;` is not a realistic example. Maybe something like the example below is better?

```php
$subdir = TEMPLATEPATH . '/subdir/';
```